registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -2,15 +2,20 @@ package org.virtuslab.ash
 
 import java.io.BufferedWriter
 import java.io.FileWriter
+import java.util.regex.PatternSyntaxException
 
 import scala.annotation.tailrec
 import scala.collection.mutable
+import scala.reflect.ClassTag
+import scala.reflect.classTag
 import scala.tools.nsc.Global
 import scala.tools.nsc.Phase
 import scala.tools.nsc.plugins.PluginComponent
 
 import org.virtuslab.ash.RegistrationCheckerCompilerPlugin.classSweepPhaseName
 import org.virtuslab.ash.RegistrationCheckerCompilerPlugin.serializerCheckPhaseName
+import org.virtuslab.ash.annotation.SerializabilityTrait
+import org.virtuslab.ash.annotation.Serializer
 
 class SerializerCheckCompilerPluginComponent(
```

Comment:
Pls publish the entire updated change on Sonatype so that I can test it against Phoenix `develop` (unless you've done so already)

Reply:
If you want to test it on Hydra, use `sbt publishLocal`. The version will be the same, as if I had published it on Sonatype.